controller/src/main/java/io/pravega/controller/store/stream/tables/EpochTransitionRecord.java

```diff
@@ -30,6 +30,7 @@
 @AllArgsConstructor
 public class EpochTransitionRecord {
     public static final EpochTransitionRecordSerializer SERIALIZER = new EpochTransitionRecordSerializer();
+    public static final EpochTransitionRecord EMPTY = new EpochTransitionRecord(Integer.MIN_VALUE, Long.MIN_VALUE, ImmutableSet.of(), ImmutableMap.of());
```

Comment:
Does it have to be `MIN_VALUE` or any negative value would do?

Reply:
We have just chosen a fixed constant and defined it as what our EMPTY record looks like. 
We could have chosen any negative value too as that would not conflict with any non-empty record.